shoop/admin/toolbar.py

```diff
@@ -109,10 +110,28 @@ class NewActionButton(URLActionButton):
 
     def __init__(self, url, **kwargs):
         kwargs.setdefault("icon", "fa fa-plus")
-        kwargs.setdefault("text", _(u"Create new\u2026"))
+        kwargs.setdefault("text", _("Create new"))
         kwargs.setdefault("extra_css_class", "btn-success")
         super(NewActionButton, self).__init__(url, **kwargs)
 
+    @classmethod
+    def for_model(cls, model, **kwargs):
+        """
+        Generate a NewActionButton for a model, auto-wiring the URL.
+
+        :param model: Model class
+        :rtype: shoop.admin.toolbar.NewActionButton|None
+        """
+
+        if "url" not in kwargs:
+            try:
+                url = get_model_url(model, kind="new")
+            except NoModelUrl:
+                return None
+            kwargs["url"] = url
+        kwargs.setdefault("text", _("New %(model)s") % {"model": model._meta.verbose_name})
```

Comment:
This is very minor thing though and this PR improves the consistency quite a bit so I won't care about this minor detail.